### PR TITLE
Make schedule visible for participants

### PR DESCRIPTION
**Pull request by DennisGranberg**
Implements logic for publishing the schedule for participants.

Adds new column "published" in events table. When set to true the schedule becomes visible.
Some minor fixes remains for a smoother user experience (eg. check if rooms exists before publishing)

### CODE REVIEW

**Comment by victoriacatalan** [comment]
Would it be impossible to have the little cards in the schedule be the ones that are clickable, and show the description? I'm thinking that from an UX perspective, having both is a little cluttered, no?

File: webapp/src/section/eventViewer/EventRegistration.tsx [hunk 1]
```diff
@@ -148,6 +147,21 @@ const EventRegistration = ({
...
+                    onChange={(e) => {
+                      handleChange(e);
+                      toggleOnlyRemote();
+                    }}
+                    name="remote"
+                  />
+                }
+                label="Distans"
+              />
+            </Box>
```
Comment (victoriacatalan) [inline webapp/src/section/eventViewer/EventRegistration.tsx:164]: This button is a duplicate, there already is one above that has the same functionality

File: webapp/src/section/eventViewer/EventViewer.tsx [hunk 2]
```diff
@@ -62,6 +64,14 @@ const EventViewer = (): ReactElement => {
 
         <RegisteredLectures lectures={filteredLectures} admin={isAdmin()} />
```
Comment (victoriacatalan) [inline webapp/src/section/eventViewer/EventViewer.tsx:65]: Is it not a little bit cluttered if we still have this showing when the schedule is published?

File: webapp/src/section/eventViewer/EventRegistration.tsx (hunk 1 above)
Comment (DennisGranberg) [reply webapp/src/section/eventViewer/EventRegistration.tsx]: Removed duplicate

File: webapp/src/section/eventViewer/EventViewer.tsx (hunk 2 above)
Comment (DennisGranberg) [reply webapp/src/section/eventViewer/EventViewer.tsx]: Now only shows for admins. Enabled button directly in the schedule for regular users to view the description.

**Comment by victoriacatalan** [comment]
Noice 🤓

**Review by victoriacatalan — APPROVED** [review]

**Review by victoriacatalan — APPROVED** [review]

**Pull request merged**